packages/fxa-auth-db-mysql/lib/db/schema/patch-097-098.sql

```diff
@@ -0,0 +1,148 @@
+--
+-- This migration introduces subscriptions enabled for accounts
+--
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('97');
+
+CREATE TABLE IF NOT EXISTS accountSubscriptions (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  uid BINARY(16) NOT NULL,
+  productName VARCHAR(191), -- TBD pending subhub docs & integration
```

Comment:
How come 191? Seems an odd choice for a length.

Reply:
Yeah, though [we're using it elsewhere](https://github.com/mozilla/fxa/search?l=PLpgSQL&q=191) - it seemed to be the magic length to avoid an error in creating too-long indexes.

Reply:
Ahh, is this an issue with UTF8MB4 perhaps?